pwa/src/templates/documentation/usageDocumentation/UsageDocumentationTemplate.module.css

```diff
@@ -11,6 +11,17 @@
   color: var(--web-app-color-text-grey) !important;
 }
 
+.codeBlock {
+  border-radius: var(--web-app-size-2xs);
+  overflow-x: scroll;
```

Comment:
Instead of `overflow-x` you could use the new logical property `overflow-inline`.